src/main/java/com/mathworks/ci/MatlabBuildStepExecution.java

```diff
@@ -114,6 +116,7 @@ private void createMatlabScriptByName(FilePath uniqueTmpFolderPath, String uniqu
         final FilePath matlabBuildFile =
                 new FilePath(uniqueTmpFolderPath, uniqueScriptName + ".m");
         final String tasks = getContext().get(EnvVars.class).expand(getTasks());
+        final String buildOptions = this.buildOptions;
```

Comment:
Does EnvVars.expand(this.buildOptions) make more sense here to support substitution

Reply:
Yes, updated!